radixdlt/babylon-wallet-android
# Handle non transferable assets on account deletion

## Description
This PR handles non transferable assets on account deletion.


## How to test

1. Try to delete an account which contains non transferable assets (for example RadQuest Hero Badge)
2. Check the dialog notifying about the present of non transferable assets is displayed
3. Check the account is successfully deleted
